### ISSUE TITLE
Add handle exceptions for exception handlers

### ISSUE THREAD

**Issue by heckad** [tiangolo/fastapi#1384]
Example of use 

```
    @app.exception_handler(errors.BaseError)
    async def handle_BaseError_error(request, exception):
        raise fastapi.exceptions.HTTPException(400, format_ccxt_errors(exception))
```

Expect fastapi return response.
Got 500 error.

It needs for unifying responses in api.

**Comment by phy25** [comment]
How do you import `errors.BaseError`? Is it from fastapi?

See also: https://fastapi.tiangolo.com/tutorial/handling-errors/#override-the-default-exception-handlers

**Comment by heckad** [comment]
> How do you import `errors.BaseError`? Is it from fastapi?

No, it's my inner error. It doesn't mean. I would like the executables in the error handler to be processed as well, with one exception. Only if it is not the same mistake.

Example invalid code
```
    @app.exception_handler(errors.BaseError)
    async def handle_BaseError_error(request, exception):
        raise exception
```

**Comment by phy25** [comment]
My two cents: I guess exception handlers are supposed to deal with its own exceptions (probably by `try...catch...`) or you will kind of get into a loop...

**Comment by tiangolo** [comment]
Thanks for the help here @phy25 ! :clap: :bow: 

If that solves the original problem, then you can close this issue @heckad  :heavy_check_mark:
